src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ProgressBar/DoubleToThicknessConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace UITests.Microsoft_UI_Xaml_Controls.ProgressBar
+{
+	internal class DoubleToThicknessConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, string language)
+		{
+			if (value is Thickness thickness)
+			{
+				return thickness;
+			}
+
+			var doubleValue = System.Convert.ToDouble(value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)